cmd/print.go

```diff
@@ -33,6 +34,9 @@ const (
 	yottabyte
 )
 
+// ErrBadArgs is a new error for when a user provides bad arguments
```

Comment:
"is a new error" will stale date pretty quickly. Don't worry about fixing this like right meow or anything, but I  try to keep my comments time-independent these days. My past mistakes in this department include references to "right now", "eventually we will", etc.